src/config.rs

```diff
@@ -50,7 +50,7 @@ use tikv_util::sys::SysQuota;
 use tikv_util::time::duration_to_sec;
 use tikv_util::yatp_pool;
 
-use crate::coprocessor_v2::Config as CoprocessorV2Config;
```

Comment:
I think this renaming is also unnecessary?

Reply:
sure